index.html

```diff
@@ -1,11 +1,42 @@
 <!DOCTYPE html>
 <html lang="en">
+
   <head>
     <link rel="stylesheet" href="styles.css">
     <link href="https://fonts.googleapis.com/css2?family=Quicksand:wght@400;700&display=swap" rel="stylesheet">
   </head>
+
   <body>
-  
+    <h1 id = "title">✨Self Care Center✨</h1>
+
+    <section class = >
+      <h2 id = "sub-title">Which type of message?</h2>
+    </section>
+
+    <form class = "radio-box">
+      <div>
+        <input type ="radio" id ="choice1" name ="radio-button" value = "shallow-thought" checked>
+        <label for ="choice1"><em>Shallow Thoughts</em></label>
+
```

Comment:
For HTML, you never really want to have indents like this. It should just be consistently stacked. So Lose the space on line: 8, 11, 15, 20, 33, 37-39

Reply:
Got it! I will make updates for the next PR.